app.js

```diff
@@ -211,19 +228,96 @@ var employeeTableBody = document.getElementById('employee-table-body');
 var tableBody;
 
 printTable();
+updateStoreOption();
 
-
+////Form Submit
 var formEl = document.getElementById('form');
 formEl.addEventListener('submit', handleSubmit);
+
 function handleSubmit(event) {
   event.preventDefault();
   console.log(event.target.store.value);
   var store = event.target.store.value;
   var min = event.target.min.value;
   var max = event.target.max.value;
   var cookieAverage = event.target.cookieAverage.value;
+  var index = -1;
 
-  var newLocation = new Location (store,min,max,cookieAverage);
+  for(var i = 0; i < locationArray.length; i++){
+    if(locationArray[i].name === store){
+      ///Update item in array and call functions again
+      console.log('Updating an already existing item');
+      ///Update Item and print table again
+      index = i;
+
+      locationArray[index].name = store;
+      locationArray[index].minCustomers = min;
+      locationArray[index].maxCustomers = max;
+      locationArray[index].cookieAverage = cookieAverage;
+
+      ///TODO: Delete table
+      // var tableSection = document.getElementById('table-section');
```

Comment:
I would just get rid of this code, if you really want to save it, just add another commit.